src/main/java/interface_adapter/club_logged_in/ClubLoggedInState.java

```diff
@@ -0,0 +1,68 @@
+package interface_adapter.club_logged_in;
+
+/**
+ * Club Logged In state.
+ */
+public class ClubLoggedInState {
+    private String clubName = "";
+
+    private String email = "";
+    private String emailError;
+
+    private String password = "";
+    private String passwordError;
+
+    public ClubLoggedInState(ClubLoggedInState copy) {
+        this.clubName = copy.clubName;
+
+        this.email = copy.email;
+        this.emailError = copy.emailError;
+
+        this.password = copy.password;
+        this.passwordError = copy.passwordError;
+    }
+
+    public ClubLoggedInState() {
+        // empty body since we have a constructor already and so no default constructor anymore.
+    }
```

Comment:
Constructors not necessary for a state; may be used for testing but creates unnecessary clutter